### PR TITLE
unnecessary variable removed

### PR DESCRIPTION
**Pull request by sanchez2210**

### CODE REVIEW

File: spec/models/inventory_spec.rb [hunk 1]
```diff
@@ -18,8 +18,8 @@
       expect(FactoryGirl.build(:inventory, name: Faker::Lorem.characters(256))).not_to be_valid
     end
 
-    it 'is invalid with a duplicate value' do 
-      inventory = FactoryGirl.create(:inventory, name: 'duplicate')
+    it 'is invalid with a duplicate value' do
+      FactoryGirl.create(:inventory, name: 'duplicate')
```
Comment (pachonk) [inline spec/models/inventory_spec.rb]: Name isn't an exact match.

Comment (sanchez2210) [inline spec/models/inventory_spec.rb]: Yes I noticed that too but I think @Exellin did that to prove uniqueness: `{ case_sensitive: false }`?

Comment (Exellin) [inline spec/models/inventory_spec.rb]: Yeah.  Although I guess it should maybe be in its own separate test?

Comment (pachonk) [inline spec/models/inventory_spec.rb]: Let's do 2 different tests like so:
```ruby
describe 'uniqueness' do
  before do
    FactoryGirl.create(:inventory, name: 'Duplicate')
  end

  it 'something 1' do
    expect(FactoryGirl.build(:inventory, name: 'Duplicate')).not_to be_valid
  end

  it 'something2' do
    expect(FactoryGirl.build(:inventory, name: 'DuPliCaTe')).not_to be_valid
  end
end
```

Comment (pachonk) [inline spec/models/inventory_spec.rb]: Can be a different PR. Just an idea.
